utils/getInvoices/tests/getInvoices.maintenance.test.ts

```diff
@@ -6,108 +6,834 @@ import { ServiceType } from '@utils/constants'
+      const company: Partial<IRealestate> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = { discount: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        discount: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 0 }, company = { totalArea: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 0,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 0 }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 0,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).toContainEqual({ 
+        type: ServiceType.Maintenance,
+        amount: 0,
+        price: 10,
+        sum: 0,
+      })
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = { totalArea: undefined }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: undefined,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
   })
 
-  it('should NOT load Maintenance from payment without maintenance', () => {
-    const company: Partial<IRealestate> = null
-    const service: Partial<IService> = null
-    const payment: Partial<IPayment> = {
-      invoice: [],
-    }
-    const prevPayment: Partial<IPayment> = null
+  describe('props: { service, company, prevPayment }', () => {
```

Comment:
There is no relation between `Maintenanca` and `prevPayment` so this test-block can be removed as unnecessary